es-core/src/renderers/Renderer.h

```diff
@@ -29,6 +30,19 @@ namespace Renderer
 
 	} // Blend::
 
+	namespace Gradient
+	{
+		enum Mode : uint8_t
+		{
+			NONE        = 0x00, // off, no gradient
+			TO_BOTTOM   = 0x01, // straight along y axis towards endcolor
+			TO_RIGHT    = 0x02, // straight along x axis towards endcolor
+			DEFAULT     = 0x04, // diagonal towards endcolor in bottom-right
```

Comment:
@Gemba I thought I had asked about this, but I might not have sent it. Shouldn't default be 0x03, if it's TO_BOTTOM and TO_RIGHT at the same time?

Reply:
Ah, I remember @pjft  but it escaped my mind. Maybe the constantnames are confusing. `TO_RIGHT` means running the gradient along x-axis only (top-right and button-right will be set to `endColor`). `TO_BOTTOM` means running the gradient along y-axis only (bottom-left and bottom-right will be set to `endColor`). In contrast `DEFAULT` sets only bottom-right to `endColor` resulting in an gradient running diagonal to screen orientation along the axis top-left to bottom-right. If you can think of more precise constants: I am open to suggestions. This is also reflected in the first two pictures I provided here. First is `DEFAULT`, second is both `TO_RIGHT` and `TO_BOTTOM` set.

Reply:
Actually, there are other options for the diagonal gradient, but this possibly one of them.